bip-tx-terminology.mediawiki

```diff
@@ -0,0 +1,291 @@
+***** '''Tap Branch''' <code>776b22a1185fb2dc9524f6b178e2693189bf01655d7f38f043923668dc5af45b</code>: The hashing partners necessary to connect the Leaf Script to the Merkle root of the Script Tree.
+** '''Transaction Header (cont)''': A collective term to refer to the meta information of a transaction, split across the transaction.
+*** '''Lock Time''' <code>ffd30a00</code>: the 4-byte lock time field, little endian for 709631
+
+===Definition of Terms===
+
+; Annex
+: [Component] A Witness Item reserved for future extensions, currently unused (see [[https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki#cite_note-5|BIP341]])
+
+; Amount
+: [Component] The count of satoshis assigned to an output
+
+; Bare Output Script
+: [Component] A Condition Script that is directly defined in the Output Script field. Used to specify all types of Bare Outputs including Pay to Pubkey, Pay to Public Key Hash, Pay to Multisig outputs, and arbitrary scripts.
+
+; Condition Script
+: [Concept] aka “locking script”. Umbrella term for scripts that encode the actual spending conditions that must be fulfilled to spend a UTXO.  Condition Scripts may be preceded by one or more forwarding scripts.  An Output Script can either directly contain a condition script in the case of a bare output, or will commit to one or multiple condition scripts for other output types.  For P2TR the tweaked public key is considered a condition script as well as each Leaf Script in the script tree.  At the time of writing, there are four distinct Condition Scripts: Witness Script, Leaf Script, Redeem Script (only non-segwit P2SH), and Bare Output Script (bare outputs, Pay to Multisig, P2PK, P2PKH).
+
+; Control Block
+: [Component] The proof of existence for a Leaf Script.  Appears as a Witness Item in Witness Stacks for P2TR script path inputs.  Only consists of the Inner Key for Script Trees with a single Leaf Script, otherwise provides the Inner Key as well as a Taproot Branch in the form of the hashing partners for the used Leaf Script.
+
+; Flag
+: [Artifact] Serialization artifact indicating features used by the transaction.  As of writing, the only allowed value is <code>01</code> which indicates that the transaction serialization has a Witness Section (see [[https://github.com/bitcoin/bips/blob/master/bip-0144.mediawiki|BIP144]]).
+
+; Forwarding Script
+: [Concept] A collective term for scripts that redirect input validation to another script or data structure.  Witness Programs and P2SH Programs are forwarding scripts.  Forwarding Scripts make use of _script templates_ that imply additional evaluation steps beyond the explicitly expressed conditions.  In the case of P2SH the Output Script in verbatim only implies that the redeemscript must be the preimage of the hash in the Output Script, but the _template_ prescribes that the redeemscript must additionally be satisfied.  For Witness Programs, the Output Script is even less verbose with more implied meaning.
```

Comment:
```suggestion
: [Concept] A collective term for scripts that redirect input validation to another script or data structure.  Witness Programs and P2SH Programs are forwarding scripts.  Forwarding Scripts make use of _script templates_ that imply additional evaluation steps beyond the explicitly expressed conditions.  In the case of P2SH the Output Script in verbatim only implies that the redeem script must be the preimage of the hash in the Output Script, but the _template_ prescribes that the redeem script must additionally be satisfied.  For Witness Programs, the Output Script is even less verbose with more implied meaning.
```

Break "redeemscript" into two words, like it is in the rest of the doc. Do you also want to capitalize it like the other occurrences?